pkg/gardenlet/apis/config/v1alpha1/types.go

```diff
@@ -350,6 +350,9 @@ type ShootCareControllerConfiguration struct {
 	// StaleExtensionHealthChecks defines the configuration of the check for stale extension health checks.
 	// +optional
 	StaleExtensionHealthChecks *StaleExtensionHealthChecks `json:"staleExtensionHealthChecks,omitempty"`
+	// ManagedResourceProgressingThreshold is the duration after which if managed resource progressing
+	// condition remains true then managed resource is considered stuck.
```

Comment:
`+optional` tag is missing.

```suggestion
	// condition remains true then managed resource is considered stuck.
	// +optional
```